playwright-e2e/pages/international-element.ts

```diff
@@ -0,0 +1,37 @@
+import { type Page, type Locator, expect } from "@playwright/test";
+import { BasePage } from "./base-page";
+
+export class InternationalElement extends BasePage {
+  readonly internationalElementHeading: Locator;
+  readonly areThereAnySuitableCarers: Locator;
+  readonly anySignificantEventsOutsideUk: Locator;
+  readonly anyIssueWithJurisdictionOfThisCase: Locator;
+  readonly awareOfAnyProceedingsOutsideTheUk: Locator;
+  readonly aGovtOrCentralAuthorityOutsideUkInvolvedInCase: Locator;
+  readonly continueButton: Locator;
+  readonly saveAndContinueButton: Locator;
+  readonly checkYourAnswers: Locator;
+
+  public constructor(page: Page) {
+    super(page);
+    this.internationalElementHeading = page.getByRole('heading', { name: 'International element' });
+    this.areThereAnySuitableCarers = page.getByRole('button', { name: 'No' });
+    this.anySignificantEventsOutsideUk = page.getByRole('button', { name: 'No' });
+    this.awareOfAnyProceedingsOutsideTheUk = page.getByRole('button', { name: 'No' });
+    this.aGovtOrCentralAuthorityOutsideUkInvolvedInCase = page.getByRole('button', { name: 'No' });
+    this.awareOfAnyProceedingsOutsideTheUk = page.getByRole('button', { name: 'No' });
+    this.continueButton = page.getByRole('button', { name: 'Continue' });
```

Comment:
these (continue button etc) are already defined in base page so these lines could be replaced and associated locators removed.
base page methods:
- await this.clickContinue();
- await this.checkYourAnsAndSubmit();